types/styled-components/index.d.ts

```diff
@@ -4,10 +4,17 @@
 //                 Ihor Chulinda <https://github.com/Igmat>
 //                 Adam Lavin <https://github.com/lavoaster>
 //                 Jessica Franco <https://github.com/Jessidhia>
+//                 Jason Killian <https://github.com/jkillian>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.9
 
-/// <reference types="node" />
+// forward declarations
+declare global {
+    namespace NodeJS {
+        // tslint:disable-next-line:no-empty-interface
+        interface ReadableStream {}
```

Comment:
out of curiosity, why this type was added here? I don't see it is used anywhere here.
(sorry for commenting merged pr)

Reply:
This was probably mistakenly added here to fix what's actually a `@types/react-native` problem instead 😔

Reply:
I see `ReadableStream` used further down in this file. It was presumably added to replace the the `/// <reference types="node" />` syntax that pulls in a dependency to the entire `node` typings.

Reply:
Thanks @minestarks, that was exactly my reasoning. The full node typings conflict with the react-native typings (see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/15960). To resolve that, I removed the reference to the node types and just stubbed out the one interface that we actually needed.

Reply:
Sorry, 🤦‍♂️